.github/workflows/analyze.yml

```diff
@@ -118,6 +118,10 @@ jobs:
     needs: analyze
     uses: ./.github/workflows/repo-checks.yml
 
+  rust-sdk:
+    needs: analyze
+    uses: ./.github/workflows/rust-sdk-tests.yml
```

Comment:
@LalitMaganti I'll restore the `if` statement here. Just trying to figure out how to run the step once to confirm it's working but not sure removing the `if` helped..

Reply:
Restored this for review. Not sure exactly what I need to do to have this step actually run at this time.